swagger.yaml

```diff
@@ -2648,6 +2648,11 @@ paths:
           type: string
           format: uuid
           required: false
+        - in: query
+          name: severity
+          description: Commma-separated event severity values or 'all' to retrive internal events in addition to user's events.
```

Comment:
I think common practice is to put the minimum level to return - for example, "info" should return info, warning, and error.

Reply:
Also, what are internal events?

Reply:
I opt for explicit API. Some clients would need only selected severity ranges that does not necessarily conform to the pattern above. This is even more evident with the new severity values such as internal and debug

Reply:
internal events are events that are used for BE-BE communication or internal service use. Immediate target are the metrics infra

Reply:
I think it works well for debug at least.  This is standard for every logging system.
This is the first time I'm hearing about internal events.  Without knowing much about it, it feels like bending the events mechanism to suit some other function, and here is a place where it breaks.

Reply:
comment resolved by moving to category